docker-compose.yml

```diff
@@ -119,7 +119,7 @@ services:
       USE_HTTPS: 0
       UWSGI_HTTP: :9000
       UWSGI_SOCKET: :9001
-      VERSION_PATH: /version.json
+      VERSION_PATH: /config/version.json
```

Comment:
Same for this as commented above on `docker-compose.lando-ui.yml`.

Reply:
I'll fix the `Dockerfiles` :)